### DISCUSSION TITLE
Multiple browsers per rule

### DISCUSSION THREAD

**Discussion by 38400000** [U-C-S/Hurl#92] [Ideas]
Hi and thanks for making this great tool.

I have been using it for some time to manage all my browser profiles. Over the years i have opened a few email addresses, and I have a browser profile for every email address. I often have 2 accounts for a specific site. So being able to choose between them is great.

A thing i have been thinking about is that i often don't remember which email (browser profile) has an account on a specific site. Which is why i have a feature request / idea regarding rulesets.
Would it be possible to have multiple browsers for a specific rule? Like "BrowserName": "Browser1", "Browser2". So if a specific link gets passed, you still get the browser picker window, but you only have the two browsers to choose between?

In any case, thanks again for making this.

**Comment by U-C-S** [comment]
Interesting Idea, Thanks for creating a discussion here.

I will try to look and make a feature out of this sometime in the future
